### PR TITLE
Fixed incorrect diagram explanation, added more clarity

### PR DESCRIPTION
**Pull request by drpayyne**
## Purpose of this pull request

This pull request (PR) fixes the incorrect diagram explanation and adds more clarity. Previous explanation was misleading due to incorrect term references and usage.

## Affected DevDocs pages
- https://devdocs.magento.com/guides/v2.3/inventory/

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

**Comment by dobooth** [comment]
Hi @drpayyne Yesterday we moved over to using the Adobe CLA. We need you to resign it before we can move on merging PRs. Thanks for the contributions!

File: guides/v2.3/inventory/index.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ The following diagram illustrates the relationship between source stocks, aggreg
 
 ![Source and aggregate stock](images/inventory-diagram-stock.png)
 
-In this diagram, a bicycle merchant has inventory for a touring bike in a warehouse and two stores. He has three stocks with configured website sales channels and sources. When a customer shops through the UK website, Magento aggregates bike inventory from the warehouse and flagship store sources for a salable quantity of 95. The touring bike can be shipped from either the warehouse or the flagship store, but not the US store. Amazon Marketplace web store same stock draws from the same aggregate stock as the UK web store.
+In this diagram, a bicycle merchant has inventory for a touring bike in two warehouses and a drop shipper. He has two stocks with configured website sales channels and sources. When a customer shops through the UK website, Magento aggregates bike inventory from the UK warehouse and the drop shipper sources, for a salable quantity of 95. The touring bike can be shipped from either the warehouse or the drop shipper, but not the NY warehouse. Amazon Marketplace has the same stock, drawing from the same aggregate stock as the UK webite.
```
Comment (keharper) [inline guides/v2.3/inventory/index.md]:
```suggestion
In this diagram, a bicycle merchant has inventory for a mountain bike in two warehouses and a drop shipper. He has two stocks with configured website sales channels and sources. When a customer shops through the UK website, Magento aggregates bike inventory from the UK warehouse and the drop shipper sources, for a salable quantity of 95. The bike can be shipped from either the warehouse or the drop shipper, but not the NY warehouse. Amazon Marketplace has the same stock, drawing from the same aggregate stock as the UK website.
```

**Comment by keharper** [comment]
Thanks @drpayyne . We'll merge this in our next batch.

**Comment by dobooth** [comment]
running tests

**Pull request merged**